test/widgets/message_list_test.dart

```diff
@@ -1755,6 +1775,64 @@ void main() {
       debugNetworkImageHttpClientProvider = null;
     });
 
+    group('User status emoji', () {
+      void checkStatusEmoji(User user, Type type, {required bool isPresent}) {
+        final nameFinder = find.text(user.fullName);
+        final statusEmojiFinder = findStatusEmoji(type);
+
+        check(nameFinder).findsOne();
+        check(statusEmojiFinder).findsExactly(isPresent ? 1 : 0);
+        if (isPresent) {
+          checkUserStatusEmoji(statusEmojiFinder, isAnimated: false);
+        }
+
+        final senderRowFinder = find.ancestor(of: nameFinder,
+          matching: find.ancestor(of: statusEmojiFinder,
+            matching: find.byType(Row)));
+        isPresent
+          ? check(senderRowFinder).findsAny()
+          : check(senderRowFinder).findsNothing();
+      }
+
+      final user1 = eg.user(userId: 1,
+        fullName: 'User with a very very very long name to check if emoji is still visible');
+      final user2 = eg.user(userId: 2, fullName: 'User 2');
```

Comment:
The specific user IDs don't matter, do they? I don't think they do. So they can be left out, to make less for the reader to have to read and think about.